### PR TITLE
create fonts.md pt-br

### PR DESCRIPTION
**Pull request by favanso**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Closes # 
- What does this PR change? it translates the fonts.md to pt-br.
- Did you change something visual? A before/after screenshot can be helpful. No

### CODE REVIEW

File: src/pages/pt-br/guides/fonts.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+layout: ~/layouts/MainLayout.astro
+setup: |
+    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+---
+
+O Astro suporta todas as estratégias mais comuns para adicionar fontes personalizadas ao projeto de seu site. Este guia lhe mostrará duas opções diferentes para incluir fontes da web em seu projeto.
+
+## Usando um arquivo de fonte local
+
+Se você quiser adicionar arquivos de fontes ao seu projeto, recomendamos adicioná-los ao seu [`public/` directory](/en/core-concepts/project-structure/#public). Em seu CSS você pode então registrar as fontes com uma declaração[`@font-face` ]([https://developer.mozilla.org/pt-BR/docs/Web/CSS/@font-face]) e usar a propriedade `font-family` para estilizar seu site.
```
Comment (delucis) [inline src/pages/pt-br/guides/fonts.md:14]: Couple of broken links here:

- internal links need to start with the current language’s tag, `/pt-br/` in this case
- looks like the `@font-face` URL got wrapped in `[]` by accident!

```suggestion
Se você quiser adicionar arquivos de fontes ao seu projeto, recomendamos adicioná-los ao seu [`public/` directory](/pt-br/core-concepts/project-structure/#public). Em seu CSS você pode então registrar as fontes com uma declaração[`@font-face` ](https://developer.mozilla.org/pt-BR/docs/Web/CSS/@font-face) e usar a propriedade `font-family` para estilizar seu site.
```

File: src/pages/pt-br/guides/fonts.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    h1 {
+      font-family: "Twinkle Star", cursive;
+    }
+    ```
+
+## Materiais extras
+
+### Adicione fontes com Tailwind
+
+Se você estiver usando a [integração Tailwind](/en/guides/integrations-guide/tailwind/), você pode adicionar uma declaração `@font-face` para uma fonte local ou usar usar a estratégia de `import` da Fontsource para registrar sua fonte. Então, siga [os documentos do Tailwind sobre como adicionar famílias de fontes personalizadas](https://tailwindcss.com/docs/font-family#using-custom-values).
```
Comment (delucis) [inline src/pages/pt-br/guides/fonts.md:110]: Same here that we should link to the `/pt-br/` guide (even if it’s not translated yet) to keep readers on their own language’s version of docs.

```suggestion
Se você estiver usando a [integração Tailwind](/pt-br/guides/integrations-guide/tailwind/), você pode adicionar uma declaração `@font-face` para uma fonte local ou usar usar a estratégia de `import` da Fontsource para registrar sua fonte. Então, siga [os documentos do Tailwind sobre como adicionar famílias de fontes personalizadas](https://tailwindcss.com/docs/font-family#using-custom-values).
```

**Review by delucis — COMMENTED** [review]
Thanks @favanso! Left two suggestions to fix broken links and will alert our other translators [on Discord](https://astro.build/chat) to check for language 🙌

**Comment by Yan-Thomas** [comment]
Thanks, @favanso! Unfortunately, I had already translated this page a few weeks ago, so I'll be closing this one in favor of #1540. Whenever you feel like translating a page I recommend taking a look at the open PRs to see if no one has translated it yet, this way we avoid duplicate effort.

**Comment by delucis** [comment]
Oh no! Sorry I missed this @Yan-Thomas 🤦 

Interesting to see the differences between the two PRs though! <https://www.diffchecker.com/7wYc8G0D>

**Pull request closed without merging**